src/OrchardCore/OrchardCore.Mvc.Core/SharedViewCompilerProvider.cs

```diff
@@ -141,13 +149,23 @@ private IViewCompiler CreateCompiler()
                 }
             }
 
-            return new SharedRazorViewCompiler(
+            return new RazorViewCompiler(
                 _fileProviderAccessor.FileProvider,
                 _razorProjectEngine,
                 _csharpCompiler,
+// See https://github.com/aspnet/Mvc/blob/release/2.2/src/Microsoft.AspNetCore.Mvc.Razor/Internal/RazorViewCompilerProvider.cs#L78 
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
The goal is to really get rid of that, it will be removed at some point.

Reply:
No, it would fail.
https://github.com/aspnet/Mvc/blob/master/src/Microsoft.AspNetCore.Mvc.Razor/Internal/RazorViewCompiler.cs#L65
They use the pragma in their master branch

Reply:
ok, good then